src/platform/w3dengine/client/w3ddisplay.cpp

```diff
@@ -40,7 +40,7 @@
 #include "network.h"
 #include "particlesysmanager.h"
 #include "physicsupdate.h"
-#include "renderdevicedesc.h"
+#include "rddesc.h"
```

Comment:
Perhaps leave `renderdevicedesc` as is? `rddesc` is not more descriptive.

Reply:
I want to stick to the original filenames and paths where we know what they are.